identity-server/src/AspNetIdentity/SecurityStampValidatorCallback.cs

```diff
@@ -9,7 +9,7 @@ namespace Duende.IdentityServer.AspNetIdentity;
 /// <summary>
 /// Implements callback for SecurityStampValidator's OnRefreshingPrincipal event.
 /// </summary>
-public class SecurityStampValidatorCallback
+public static class SecurityStampValidatorCallback
```

Comment:
Is there any chance or good reason that someone would derive from this callback and add new behavior?